src/Bicep.Core/Registry/ContainerRegistryClientFactory.cs

```diff
@@ -14,7 +14,7 @@ public BicepRegistryBlobClient CreateBlobClient(Uri registryUri, string reposito
             var options = new ContainerRegistryClientOptions();
 
             // ensure User-Agent mentions us
-            options.Diagnostics.ApplicationId = $"{LanguageConstants.LanguageId} {ThisAssembly.AssemblyFileVersion}";
+            options.Diagnostics.ApplicationId = $"{LanguageConstants.LanguageId}/{ThisAssembly.AssemblyFileVersion}";
```

Comment:
>/ [](http://example.com/codeflow?start=80&length=1)

"bicep" and the version landed as separate components in the user agent header without the `/`